README.md

```diff
@@ -2,11 +2,13 @@
 
 [![CircleCI](https://circleci.com/gh/fullstorydev/fullstory-babel-plugin-react-native.svg?style=svg)](https://circleci.com/gh/fullstorydev/fullstory-babel-plugin-react-native)
 
-FullStory's React Native babel plugin performs compile-time transformations to allow FullStory to access certain React Native attributes.
+FullStory's React Native babel plugin performs compile-time transformations to add FullStory-specific attributes to all React Native controls. This plugin should only be used in conjunction with `@fullstory/react-native`.
 
```

Comment:
We've typically added a "This is a WIP" message to the README until the package is deployed on NPM.

Reply:
done